### PR TITLE
React to Options/Rename Configure => Add

### PR DESCRIPTION
**Pull request by HaoK**
- Overload for UseCookie that takes IOptions instance (for identity/multi cookie usage)
- Rename Configure methods => Add for consistency

@Tratcher

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "projects": ["src"]
+    "projects": ["src", "C:\\Github\\Options\\src", "C:\\Github\\HttpAbstractions\\src", "C:\\Github\\DataProtection\\src"]
```
Comment (HaoK) [inline global.json]: Will revert on squash

**Comment by HaoK** [comment]
Fixes https://github.com/aspnet/Security/issues/417

File: src/Microsoft.AspNet.Authentication.OAuthBearer/OAuthBearerServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+using Microsoft.AspNet.Authentication.OAuthBearer;
+using Microsoft.Framework.Configuration;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods to add OAuth Bearer authentication capabilities to an HTTP application pipeline
+    /// </summary>
+    public static class OAuthBearerServiceCollectionExtensions
```
Comment (Tratcher) [inline src/Microsoft.AspNet.Authentication.OAuthBearer/OAuthBearerServiceCollectionExtensions.cs]: Leftover file.

**Comment by Tratcher** [comment]
:shipit:

**Comment by HaoK** [comment]
bf2b771eabcccfd6317e50366d4869f66090b23c

**Pull request closed without merging**